include/deal.II/base/vectorization.h

```diff
@@ -5439,6 +5448,19 @@ namespace std
     return x.get_min(y);
   }
 
+
+
+  /**
+   * Iterator traits for VectorizedArrayIterator.
+   */
+  template <class T>
+  struct iterator_traits<dealii::VectorizedArrayIterator<T>>
+  {
+    using iterator_category = forward_iterator_tag;
```

Comment:
Not wrong, but don't you in fact have a random access iterator? (That might require you to implement `operator-` and possibly `operator +=/-=`, but that would enable a more efficient implementation of `std::distance`.)

Reply:
Thanks for the hint @bangerth! Let me try that out!

Reply:
Done :)